Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/BaseCursor.cs

```diff
@@ -361,16 +361,19 @@ protected virtual void Start()
 
         private void Update()
         {
-            if (!CoreServices.InputSystem.FocusProvider.TryGetFocusDetails(Pointer, out focusDetails))
+            if ((CoreServices.InputSystem != null) && (CoreServices.InputSystem.FocusProvider != null))
```

Comment:
I think this may no longer be needed as I think @davidkline-ms  has fixed issues regarding switching mrtk profiles at runtime.